axlearn/common/flash_attention/layer.py

```diff
@@ -48,8 +48,10 @@ class Config(GroupedQueryAttention.Config):
         causal: bool = False
         # The block size used to tile attention computation (for TPU only).
         # Should be less than the target sequence length and a multiple of 128 on TPU.
-        # TODO(tom_gunter): Expose GPU block-size (currently always 128) & unify.
         tpu_block_size: int = 512
+        # The default GPU block-size of 128 works on most accelerators
+        # NVIDIA Blackwell (B200) requires a smaller block-size
+        gpu_block_size: int = 128
```

Comment:
```suggestion
        gpu_block_size: Optional[int] = None
```
To avoid many golden config changes.